code/game/machinery/autolathe/autolathe.dm

```diff
@@ -290,6 +290,9 @@
 
 	storage_capacity[DEFAULT_WALL_MATERIAL] = mb_rating * 25000
 	storage_capacity[MATERIAL_GLASS] = mb_rating * 12500
+	storage_capacity[MATERIAL_ALUMINIUM] = mb_rating * 25000
+	storage_capacity[MATERIAL_PLASTIC] = mb_rating * 12500
+	storage_capacity[MATERIAL_LEAD] = mb_rating * 12500
```

Comment:
Did you test that this works as in game without editing the TGUI?

Reply:
Can confirm it does:
![image](https://github.com/Aurorastation/Aurora.3/assets/1779662/994746d3-e8e8-4b7c-a980-95b02518d131)